PublicDateAtUSN: 2020-01-13 19:15:00 UTC
Candidate: CVE-2020-5390
PublicDate: 2020-01-13 19:15:00 UTC
References:
 https://github.com/IdentityPython/pysaml2/commit/5e9d5acbcd8ae45c4e736ac521fd2df5b1c62e25 (v5.0.0)
 https://github.com/IdentityPython/pysaml2/commit/f27c7e7a7010f83380566a219fd6a290a00f2b6e
 https://github.com/IdentityPython/pysaml2/releases
 https://github.com/IdentityPython/pysaml2/releases/tag/v5.0.0
 https://pypi.org/project/pysaml2/5.0.0/
 https://ubuntu.com/security/notices/USN-4245-1
 https://www.cve.org/CVERecord?id=CVE-2020-5390
Description:
 PySAML2 before 5.0.0 does not check that the signature in a SAML document
 is enveloped and thus signature wrapping is effective, i.e., it is affected
 by XML Signature Wrapping (XSW). The signature information and the
 node/object that is signed can be in different places and thus the
 signature verification will succeed, but the wrong data will be used. This
 specifically affects the verification of assertion that have been signed.
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
Priority: medium
Discovered-by:
Assigned-to:
CVSS:
 nvd: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N [7.5 HIGH]


Patches_python-pysaml2:
upstream_python-pysaml2: needs-triage
precise/esm_python-pysaml2: DNE
trusty_python-pysaml2: ignored (end of standard support)
trusty/esm_python-pysaml2: DNE
esm-infra-legacy/trusty_python-pysaml2: DNE
xenial_python-pysaml2: released (3.0.0-3ubuntu1.16.04.4)
esm-infra/xenial_python-pysaml2: released (3.0.0-3ubuntu1.16.04.4)
bionic_python-pysaml2: released (4.0.2-0ubuntu3.1)
esm-infra/bionic_python-pysaml2: released (4.0.2-0ubuntu3.1)
disco_python-pysaml2: released (4.5.0+dfsg1-0ubuntu2.19.04.1)
eoan_python-pysaml2: released (4.5.0+dfsg1-0ubuntu2.19.10.1)
devel_python-pysaml2: released (4.9.0-0ubuntu2)
